api/src/test/java/uk/gov/dhsc/htbhf/eligibility/helper/PersonTestFactory.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.dhsc.htbhf.eligibility.helper;
+
+import uk.gov.dhsc.htbhf.eligibility.model.AddressDTO;
+import uk.gov.dhsc.htbhf.eligibility.model.PersonDTO;
+
+import java.time.LocalDate;
+
+public class PersonTestFactory {
```

Comment:
I'd call this `PersonDTOTestDataFactory` as we have more than one person model object in this project.

Reply:
Renamed.